### PR TITLE
Create consistent design (fixes #769)

### LINKED ISSUE #769: Inconsistent design
Some of button are using white text and some black.
I understand we have black text over white button but some button even if it does not have white button are black text.

Disabled buttons should be greyed or black. Button with action working should be white (unless it has white background)

### PR DESCRIPTION
**Pull request by zelmi**
Fixes #769 

## Description
Changes the color of a button's text to white if the button has a non-white background, to create design consistency.

#### Changes this
Screen Shot 2019-04-30 at 4 42 29 PM

#### To this
Screen Shot 2019-04-30 at 4 43 47 PM

`Note: This inconsistency in text color occurred in many files. `

### CODE REVIEW

**Comment by zelmi** [comment]
Thanks @leomaxi . Should I make the background a darker shade of gray to make the text more visible, or should I return the text color to black?

**Comment by leomaxi** [comment]
I think you should bake the background a darker shade of gray

**Comment by zelmi** [comment]
I've changed all the gray buttons to be darker, making text more visible:

#### Changed this: 

Screen Shot 2019-05-01 at 4 46 40 PM

#### To this: 

Screen Shot 2019-05-01 at 4 47 46 PM

I've also added a margin of 10dp between any two grey buttons, because there would be no visible disconnection between them when a darker background was used.

#### Before adding the margin

Screen Shot 2019-05-01 at 4 40 59 PM

#### After adding the margin

Screen Shot 2019-05-01 at 5 08 02 PM

File: app/src/main/res/layout-large-land/activity_login.xml [hunk 1]
```diff
@@ -188,12 +188,13 @@
...
+                        android:layout_height="wrap_content"
                         android:layout_gravity="center"
                         android:layout_margin="@dimen/padding_normal"
-                        android:theme="@style/PrimaryFlatButton"
-                        android:id="@+id/become_member"
+                        android:background="@color/md_grey_500"
                         android:text="Become a member"
-                        android:layout_height="wrap_content" />
+                        android:textColor="@color/md_white_1000"
+                        android:theme="@style/PrimaryFlatButton" />
```
Comment (rrijal53) [inline app/src/main/res/layout-large-land/activity_login.xml]: Would be better if you create a button style in styles.xml and apply that style in all the button. 
It helps to eliminate redundancy in xml. You can do that in different PR .

Comment (leomaxi) [reply app/src/main/res/layout-large-land/activity_login.xml]: New Fix https://github.com/open-learning-exchange/myplanet/pull/812
